### PR TITLE
Add image alignment research code

### PR DESCRIPTION
**Pull request by jonchang**
# Pull Request

## Description
Adds research code related to the image alignment spike. 

## Related Issues
https://github.com/CDCgov/IDWA/issues/7

## Additional Notes
https://docs.google.com/document/d/12AibUgv1V7n3oiO5RQZxHE-PTrjKyHrsAQ6pVlar9i8/edit?usp=drive_link

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [x] The code follows best practices and conventions.
- [x] The changes implement the desired functionality or fix the reported issue.
- [x] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: OCR/image-alignment/go.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# download pertussis PDFs from:
+# https://drive.google.com/drive/folders/1vG-XwFiOoRunx9eXCmd41WpAI5NCizeM
+
+set -eu
+
+# Extract the first page of each PDF and rasterize it as a PNG
+convert -density 300 "0_Pertussis.fillable_Redacted.pdf[0]" -background white -alpha remove pertussis-filled.png
```
Comment (zdeveloper) [inline OCR/image-alignment/go.sh:8]: which convert package is this?

**Review by zdeveloper — CHANGES_REQUESTED** [review]
I think this is a great start, I am just a bit unsure on how this will fit in with the rest of the application and how we will align an input image to a template considering we are just inputting one file

**Review by zdeveloper — DISMISSED** [review]
Thank you for the changes

**Review by arinkulshi-skylight — APPROVED** [review]
LGTM
